src/test/java/oracle/kubernetes/operator/create/CreateDomainInputsValidationTest.java

```diff
@@ -466,6 +493,10 @@ private String invalidEnumParamValueError(String param, String val) {
     return errorRegexp("Invalid.*" + param + ".*" + val);
   }
 
+  private String invalidRelatedParamValueError(String param, String val, String param2, String val2) {
+    return errorRegexp("Invalid.*" + param + ".*" + val + " with " + param2 + ".*" + val2);
+  }
+
   private String paramMissingError(String param) {
     return errorRegexp(param + ".*missing");
   }
```

Comment:
Please update the md files too.

Reply:
Will these changes require that customers regenerate their domains?  If so, it should be listed in site/recent-changes.md

Reply:
No, these changes do not require customers to regenerate their domains unless they want to specify a non-default setting for the reclaim policy.

Reply:
OK - the style I've been going with is to use upper case for our enums only - pass thru enums use the case of whatever you're passing through to (e.g. IfNotPresent / Always / Never)

Reply:
Okay. I'll keep the enum as it is. Thanks.